diesel/src/macros/associations/belongs_to.rs

```diff
@@ -45,15 +45,25 @@
 ///
 /// [custom_derive]: https://crates.io/crates/custom_derive
 ///
-/// ```ignore
+/// ```no_run
+/// # #[macro_use] extern crate diesel;
+/// # #[macro_use] extern crate custom_derive;
+/// # table! { users { id -> Integer, } }
+/// # table! { posts { id -> Integer, user_id -> Integer, } }
+/// # custom_derive!{
+/// #     #[derive(Identifiable(users))]
+/// #     pub struct User {
+/// #         id: i32,
+/// #     }
+/// # }
 /// custom_derive! {
-///     #[derive(BelongsTo(User, foreign_key = user_id)]
-///     #[table_name(posts)]
+///     #[derive(BelongsTo(User, foreign_key = user_id, table = posts), Identifiable(posts))]
```

Comment:
The goal is to provide a single consistent API and for the macro to be useable with or without custom_derive